language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/BallerinaLanguageServer.java

```diff
@@ -202,6 +217,9 @@ public ExtendedLanguageClient getClient() {
     public void initialized(InitializedParams params) {
         LSClientLogger clientLogger = LSClientLogger.getInstance(this.serverContext);
         clientLogger.logMessage("LS offline source compilation set to " + CommonUtil.COMPILE_OFFLINE);
+
+        registerBalaUriScheme();
```

Comment:
We register the hover, references and definition features for `.bal` files with the `file` uri scheme, within this method and it seems bit out of sync with the name. Shall we try to reorganise this for the sake of readability?

Reply:
Yes. Makes sense.

Reply:
Done. Can you have a look. I refactored the dynamic command registration parts to separate methods. Check and let me know if anything needs to be changed.